app/Http/Resources/CityWithoutAssignedCoordinatesResources.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class CityWithoutAssignedCoordinatesResources extends JsonResource
```

Comment:
```suggestion
class CityWithoutAssignedCoordinatesResource extends JsonResource
```

Reply:
I think that singular "resource" will be better

Reply:
changed